packages/web-app/lib/constants/sepolia.tsx

```diff
@@ -0,0 +1,10 @@
+import { BigNumber } from "ethers";
+
+export const permit2Address = "0x000000000022D473030F116dDEE9F6B43aC78BA3";
```

Comment:
N00b question:  What is "permit2"?

Reply:
Since our smart contract needs it for the constructor function, I can dive into this app to see if we need this address here or not. Here is the definition of permit2: https://docs.uniswap.org/contracts/permit2/overview